web_ui/frontend/app/cache/page.tsx

```diff
@@ -0,0 +1,142 @@
+                                            },
+                                        },
+                                    },
+                                }}
+                                datasetOptions={[
+                                    {label: "Total Storage (Bytes)", borderColor: "#000000"},
+                                    {label: "Free Storage (Bytes)", borderColor: "#54ff80"}
+                                ]}
+                            />
+                        </Box>
+                    </Box>
+                </Grid>
+                <Grid item xs={12} lg={6}>
+                    <Box sx={{backgroundColor: "#F6F6F6", borderRadius: "1rem"}} p={2}>
+                        <Typography variant="h4" mb={1}>Transfer</Typography>
+                        <Box minHeight={"200px"}>
+                            <RateGraph
+                                rate={TimeDuration.fromString("3h")}
+                                duration={TimeDuration.fromString("7d")}
+                                resolution={TimeDuration.fromString("3h")}
+                                metrics={['xrootd_server_bytes{direction="rx"}', 'xrootd_server_bytes{direction="tx"}']}
+                                boxProps={{
+                                    maxHeight:"400px",
+                                    flexGrow:1,
+                                    justifyContent:"center",
+                                    display:"flex",
+                                    bgcolor:"white",
+                                    borderRadius:2
+                                }}
+                                options={{
+                                    scales: {
+                                        x: {
+                                            type: 'time',
+                                            time: {
+                                                round: 'second',
+                                            }
+                                        }
+                                    },
+                                    plugins: {
+                                        zoom: {
+                                            zoom: {
+                                                drag: {
+                                                    enabled: true,
+                                                },
+                                                mode: 'x',
+                                                // TODO - Implement smart update on zoom: onZoom: (chart) => console.log(chart)
+                                            },
+                                        },
+                                    },
+                                }}
+                                datasetOptions={[
+                                    {label: "Bytes Received", borderColor: "#0071ff"},
```

Comment:
```suggestion
                                    {label: "Bytes Received (Bps)", borderColor: "#0071ff"},
```

Reply:
@haoming29 Are you sure? Since I expose the resolution the timeframe will be whatever value they choose.

Reply:
According to Prometheus documentation:

> rate(v range-vector) calculates the per-second average rate of increase of the time series in the range vector.

What I interpret out of that is `rate()` gives bytes per second. The resolution we are setting is the timeline resolution for the returned `rate()` data points, if that makes sense

Reply:
And the `range-vector` essentially select the range of data to calculate the average, so it's like "Bps in this given period of time?"

Reply:
Ah yes, thank you. That explains why the rates didn't half when I turned the resolution to 2.